ext/SpeedyWeatherGeoMakieExt/SpeedyWeatherGeoMakieExt.jl

```diff
@@ -0,0 +1,59 @@
+module SpeedyWeatherGeoMakieExt
+
+using SpeedyWeather
+using GeoMakie
+
+include("faces.jl")
+
+SpeedyWeather.globe(Grid::Type{<:AbstractGridArray}, nlat_half::Integer) = SpeedyWeather.globe(SpectralGrid(; Grid, nlat_half))
+SpeedyWeather.globe(SG::SpectralGrid) = globe(Geometry(SG))
+
+function SpeedyWeather.globe(geometry::Geometry{NF, Grid}) where {NF, Grid}
+
+    faces, facesr = _faces(geometry)
+
+    transf = GeoMakie.Geodesy.ECEFfromLLA(GeoMakie.Geodesy.WGS84())
+
+    fig = Figure(size=(800, 800));
+    ax = LScene(fig[1,1], show_axis=false);
+
+    # background image
+    bg = meshimage!(ax, -180..180, -90..90, rotr90(GeoMakie.earth()); npoints = 100, z_level = -10_000);
+    bg.transformation.transform_func[] = transf
+
+    # cell centers
+    color = :black
+    s = scatter!(ax, geometry.londs, geometry.latds, markersize=5; color)
+    s.transformation.transform_func[] = transf
+
+    # cell faces
+    for i in 1:geometry.nlon_max÷4
+        for offset in [0, 90, 180, 270]
+            lp1 = lines!(ax, facesr[i, :, 1] .+ offset, facesr[i, :, 2]; color)
+            lp2 = lines!(ax, offset .- faces[i, :, 1], faces[i, :, 2]; color)
+
+            lp1.transformation.transform_func[] = transf
+            lp2.transformation.transform_func[] = transf
+        end
+    end
```

Comment:
This will get really slow for large numbers of lines, you should consider building up a vector of `Point2` or so that you can feed to a single `lines` call!

Reply:
And non-connectivity of lines is then done via NaNs?

Reply:
Yep

Reply:
Rendering might be a bit faster, but this is primarily because each lines plot compiles a new shader!  So at some density you will overload your GPU